### PR TITLE
LPS-68180 Update build script rules

### PR DESCRIPTION
**Pull request by csierra**
/cc @topolik Test is not passing because antisamy formats the output in a different way than the test expects

### CODE REVIEW

**Comment by csierra** [comment]
hey @ithildir, could you please have a look at this pull? I did what you suggested of keeping the file in git and execute the download task only when the file is missing. I have a problem, though, and it is that source formatter tries to format source the file and it fails miserably. 

Is there any way to prevent source formatter from _"fixing"_ this configuration XML?

Thanks.

**Comment by csierra** [comment]
ci:stop

**Comment by Ithildir** [comment]
Hi @csierra!

Have you tried to add a `source.formatter.excludes` line in this [`source-formatter.properties`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/foundation/portal-security/source-formatter.properties)?

**Comment by csierra** [comment]
hey @Ithildir... no... but that's global, isn't it? don't we have anything local to the module?

**Comment by Ithildir** [comment]
I've seen that @hhuijser changed things lately, it seems that modules (or groups of modules like `portal-security`) can have their own `source-formatter.properties`. For example [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/collaboration/blogs/source-formatter.properties#L9).

**Comment by csierra** [comment]
hey... that's cool but are the paths still relative to the root of the portal instead of the module?

**Comment by csierra** [comment]
I tried both in `portal-security` and `portal-security-antisamy` with no luck. gradlew formatSource is still failing :-(

**Comment by Ithildir** [comment]
Then we need to summon @hhuijser to shine a light on this...

**Comment by csierra** [comment]
summon summon

**Comment by hhuijser** [comment]
@csierra it worked for me: https://github.com/csierra/liferay-portal/pull/250

If you keep having problems, I'd suggest rebasing to latest master and making sure you're running latest SF.

**Comment by ambrinchaudhary** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/foundation/portal-security/portal-security-antisamy/src/main/resources/META-INF/resources/sanitizer-configuration.xml [hunk 1]
```diff
@@ -488,6 +488,25 @@ http://www.w3.org/TR/html401/struct/global.html
...
+				</regexp-list>
+			</attribute>
+			<attribute name="media">
+				<regexp-list>
+					<regexp name="anything" />
+				</regexp-list>
+			</attribute>
+			<attribute name="type">
+				<regexp-list>
+					<regexp value="[a-zA-Z0-9]+/[a-zA-Z0-9]+" />
```
Comment (topolik) [inline modules/apps/foundation/portal-security/portal-security-antisamy/src/main/resources/META-INF/resources/sanitizer-configuration.xml]: @csierra Just out of curiosity, this is a mime type regex, correct? Mime types contain also dashes. Or is it any intent in leaving the dash outside? Thx!

**Comment by ambrinchaudhary** [comment]
This was the new PR: https://github.com/ambrinchaudhary/liferay-portal/pull/102

**Pull request closed without merging**